FeatureExtraction/Wavelets/wavelets_features.py

```diff
@@ -24,29 +24,36 @@ def get_wavelet_features(image: GreyscaleImage, mother_wavelet: str) -> dict:
             "Image should be in greyscale, use GreyscaleImage type")
     coeffs = pywt.dwt2(image, mother_wavelet)
     cA, (cH, cV, cD) = coeffs
-    features_dict = {}
+    features_dict = OrderedDict()
     for detail_coefficients, name in zip([cH, cV, cD], ["cH", "cV", "cD"]):
-        max_value, avg_value, kurtosis, skewness = get_features_for_detail_coefficients(
-            detail_coefficients)
-        features = {mother_wavelet + "_" + name + "_max": max_value,
-                    mother_wavelet + "_" + name + "_avg": avg_value,
-                    mother_wavelet + "_" + name + "_kurtosis": kurtosis,
-                    mother_wavelet + "_" + name + "_skewness": skewness}
-        features_dict = {**features_dict, **features}
+        detail_features = get_features_for_detail_coefficients(
+            detail_coefficients, mother_wavelet, name)
+        features_dict = {**features_dict, **detail_features}
```

Comment:
I think this is no longer OrderedDict after such operation.

Reply:
+1. Maybe a test?

Reply:
Done